src/commands/RunCommand.js

```diff
@@ -56,9 +56,8 @@ export default class RunCommand extends Command {
     NpmUtilities.runScriptInDir(this.script, this.args, pkg.location, (err, stdout) => {
       if (err) {
         this.logger.error(`Errored while running npm script '${this.script}' in '${pkg.name}'`, err);
-      } else {
-        this.logger.info(stdout);
       }
+      this.logger.info(stdout);
```

Comment:
Maybe better to put the output from `stdout` _before_ the error output, so that the errors are more likely to remain on-screen after exit?